python/ray/dag/compiled_dag_node.py

```diff
@@ -1766,9 +1788,12 @@ def teardown(self, wait: bool):
                     _destroy_nccl_group(outer._nccl_group_id)
 
                 if wait:
-                    logger.info("Waiting for worker tasks to exit")
+                    logger.debug("Waiting for worker tasks to exit")
                     self.wait_teardown()
-                    logger.info("Teardown complete")
+                    logger.debug("Teardown complete")
```

Comment:
Should we keep one of these messages as INFO for now?

Reply:
I want to disable it before beta release. But yeah maybe good idea to keep it until last minute